packages/babel-plugin-proposal-decorators/test/fixtures/2022-03-accessors/undecorated-public/output.js

```diff
@@ -15,10 +17,10 @@ class Foo {
     this.#B = v;
   }
   #C = 456;
-  get ['c']() {
+  get [_computedKey]() {
     return this.#C;
   }
-  set ['c'](v) {
+  set [_computedKey](v) {
```

Comment:
This test captured the bug that computed key in a non-decorated accessor is not properly memoised.